webpack.config.dev.js

```diff
@@ -50,6 +60,5 @@ export default {
       {test: /(\.css|\.scss)$/, loaders: ['style', 'css?sourceMap', 'postcss', 'sass?sourceMap']},
```

Comment:
Heard of others having trouble with webpack 2 and the loader shorthand (sans `-loader` suffix) and having to add the `-loader` suffix everywhere. Possible that's been resolved since then, though.